### ISSUE TITLE
"What a blast" questbook issue

### ISSUE THREAD

**Issue by vevlaa** [adam9899/Issues#574]
The quest in Introduction to science will not detect the Industrial Blast Furnace from the inventory. 
Standard machine casings are detected, but the furnace will not be detected. 
I have tired making the machine a few times, but each time, the result are the same.

This issue is preventing me from completing the introduction to science chapter, and are unable to complete stuff that require this chapter to be done.

Pack version is 1.3.5.3. Server was updated from 1.3.5.2 a few days after 5.3 was released.

**Comment by vevlaa** [comment]
After looking a bit deeper into the recipe, I notice the Iron furnace needed to construct it have a duplicate recipe with Iron plate from Quark Could this be related?

**Comment by vevlaa** [comment]
Okay I got it solved, I had to make it in a vanilla crafting table, and then make the whole furnace multi-block (all made from a vanilla table) with two lava source-blocks inside. Then it recognised it. 
The initial was made using advanced storage remote, and that was the one with the issue. 
I broke the multiblock and had both furnaces stack in the inventory afterwards. (they both have 0 NBT tags)

**Comment by Avalonzo** [comment]
FTBQuests is very specific as what items it needs for quest completions and I was wondering if it needed crafting in a non storage system crafting table #541 and it seems that way.

**Comment by JathTyki** [comment]
I am actually having this exact same issue, and creating everything from a vanilla crafting table didn't seem to help. I can't get it to recognize that I have made a blast furnace at all.

**Comment by vevlaa** [comment]
> I am actually having this exact same issue, and creating everything from a vanilla crafting table didn't seem to help. I can't get it to recognize that I have made a blast furnace at all.

Did you try building the multiblock? I had to build the whole thing (with lava inside) and added the furnace part last.

**Comment by Avalonzo** [comment]
Have you tried crafting it in a tinkers crafting station as that's what worked for me. I also noticed you both mention using a vanilla crafting table but the vanilla crafting table is disabled in this pack and is instead replaced with the crafting table from the fast work bench [mod](https://www.curseforge.com/minecraft/mc-mods/fastworkbench) and maybe that is also causing the same issue as crafting in a storage system. If that don't work try what Vevlaa suggested.

**Comment by JathTyki** [comment]
@vevlaa yep,, I did build the multi block completely, lava included. Didnt change the quest completion unfortunately. Created them 3 times, 3 different ways (the items that is).

@Avalonzo I actually did try the tinkers one. I tried the ME system, tinkers, then vanilla (fastworkbench) one, in that order. Unfortunately the quest didnt pick up. :(

**Comment by Avalonzo** [comment]
If you want to set it as complete yourself enter cheat mode and type "/ftbquests editing_mode" without the " then right click the quest and click complete instantly then just type "/ftbquests editing_mode" again to exit edit mode.

**Comment by asoshinsky** [comment]
everyone on our server has the same issue
